src/components/sticky-object.js

```diff
@@ -88,10 +88,11 @@ AFRAME.registerComponent("sticky-object-zone", {
     getLastWorldPosition(this.el.object3D, this.worldPosition);
     getLastWorldQuaternion(this.el.object3D, this.worldQuaternion);
 
-    const dir = new THREE.Vector3(0, 0, 5).applyQuaternion(this.el.object3D.quaternion);
-    this.bootImpulsePosition = new CANNON.Vec3(0, 0, 0);
-    this.bootImpulse = new CANNON.Vec3();
-    this.bootImpulse.copy(dir);
+    //TODO: support this with ammo
```

Comment:
Should this be removed or does this still need to be done?

Reply:
This still needs to be done, but I didn't do it because none of this code is live right now.